core/settings/qPrefCloudStorage.cpp

```diff
@@ -7,7 +7,6 @@ static const QString group = QStringLiteral("CloudStorage");
 qPrefCloudStorage::qPrefCloudStorage(QObject *parent) : QObject(parent)
 {
 }
-
```

Comment:
that seems like a pointless whitespace change... we typically do have an empty line before the beginning of a new function... not worth redoing things over, so just an idle nag.